test/framework/alertmanager.go

```diff
@@ -255,6 +256,27 @@ func (f *Framework) CreateSilence(ns, n string) (string, error) {
 	return createSilenceResponse.Data.SilenceID, nil
 }
 
+func (f *Framework) SendAlertToAlertmanager(ns, n string) error {
+	var postAlertResp amAPIPostAlertResp
+	request := ProxyPostPod(f.KubeClient, ns, n, "web", "api/v1/alerts", `
+[{"labels":{"alertname":"ExampleAlert","prometheus":"my-prometheus"},"annotations":{},"startsAt":"2018-09-17T16:27:45.70751247Z","endsAt":"2030-09-17T16:31:45.80751247Z","generatorURL":"http://prometheus-test-0:9090/graph?g0.expr=vector%281%29\u0026g0.tab=1"}]
```

Comment:
let's not hardcode the dates here, even if it's in 12 years

Reply:
Agree, that approach is a bit hacky. Updated.